flycheck-clj-kondo.el

```diff
@@ -86,18 +86,20 @@ Argument EXTRA-ARGS: passes extra args to the checker."
 
                    ;; Else use the mode to infer which language to turn on.
                    (pcase ,lang
-                     ("clj" `(equal 'clojure-mode major-mode))
-                     ("cljs" `(equal 'clojurescript-mode major-mode))
-                     ("cljc" `(equal 'clojurec-mode major-mode)))))))
+                     ("clj"  (member major-mode '(clojure-mode clojure-ts-mode)))
+                     ("cljs" (member major-mode '(clojurescript-mode clojurescritps-ts-mode)))
+                     ("cljc" (member major-mode '(clojurec-mode clojurec-ts-mode))))))))
+
```

Comment:
Note, I don't fully understand what this code intended to do. It seems it is doing the same as the `:modes` argument?

Reply:
I don't know